pypeit/core/extract.py

```diff
@@ -1656,7 +1673,10 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, maxdev
     skymask_objflux[thismask] = np.interp(ximg[thismask],xtmp,qobj) < (skymask_nthresh*threshold)
     # Still have to make the skymask
     skymask_fwhm = create_skymask_fwhm(sobjs,thismask)
-    skymask = skymask_objflux | skymask_fwhm
+    if boxcar_rad_pix is None:
+        skymask = skymask_objflux | skymask_fwhm
+    else:  # Enforces boxcar radius masking
```

Comment:
skymask =  skymask_objflux | skymask_fwhm if boxcar_rad_pix is None else skymask_objflux & skymask_fwhm

Reply:
Is there a reason why you didn't implement this simpler logic?

Reply:
I prefer it this way.